ajax/libs/porthole/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "porthole",
+  "version": "1.0.0",
+  "filename": "porthole.min.js",
+  "description": "A proxy to safely communicate to cross-domain iframes in JavaScript.",
+  "homepage": "https://ternarylabs.github.com/porthole/",
```

Comment:
https://ternarylabs.github.com/porthole/ will redirect to http://ternarylabs.github.io/porthole/. I think let's use https://ternarylabs.github.io/porthole/ here.